core/src/main/scala/org/bitcoins/core/crypto/ExtKey.scala

```diff
@@ -180,48 +221,59 @@ sealed abstract class ExtPublicKey extends ExtKey {
 
   final override def deriveChildPubKey(idx: UInt32): Try[ExtPublicKey] = {
     if (idx >= ExtKey.hardenedIdx) {
-      Failure(new IllegalArgumentException("Cannot derive hardened child from extended public key"))
+      Failure(
+        new IllegalArgumentException(
+          "Cannot derive hardened child from extended public key"))
     } else {
-      val data = key.bytes ++ idx.bytes
-      val hmac = CryptoUtil.hmac512(chainCode.bytes, data)
+      val data     = key.bytes ++ idx.bytes
+      val hmac     = CryptoUtil.hmac512(chainCode.bytes, data)
       val (il, ir) = hmac.splitAt(32)
-      val priv = ECPrivateKey(il)
-      val tweaked = NativeSecp256k1.pubKeyTweakAdd(
-        key.bytes.toArray,
-        hmac.toArray,
-        priv.isCompressed)
+      val priv     = ECPrivateKey(il)
+      val tweaked = NativeSecp256k1.pubKeyTweakAdd(key.bytes.toArray,
```

Comment:
Is there a conflict which doesn't allow this `=` to be moved over one character because this would create a line longer than 80 chars?